homeassistant/components/tellduslive.py

```diff
@@ -56,7 +57,8 @@ def setup(hass, config):
         return False
 
     hass.data[DOMAIN] = client
-    client.update(utcnow())
+
+    hass.bus.listen(EVENT_HOMEASSISTANT_START, client.update)
```

Comment:
Instead of this, move `self._client.entities.append(self)` from TelldusEntity `__init__` to a new method on that class:

```python
@asyncio.coroutine
def async_added_to_hass(self):
    """Entity has been added to hass."""
    self._client.entities.append(self)
```

When async_added_to_hass gets called, `hass` is set so updating the client will not result in entities without `hass`